imports/plugins/core/orders/client/components/orderDashboard.js

```diff
@@ -18,6 +18,7 @@ class OrderDashboard extends Component {
     handleSelect: PropTypes.func,
     isLoading: PropTypes.object,
     multipleSelect: PropTypes.bool,
+    onDatesChange: PropTypes.func,
```

Comment:
Is there some sort of pattern you are using for naming functions `onThisChange` as opposed to `handleThisChange`? If not I would like to standardize on one or the other. If so, let's codify this in a style guide somewhere so we can be consistent.